src/lib.rs

```diff
@@ -1219,7 +1219,7 @@ pub fn z85_encode(data: &[u8]) -> result::Result<String, EncodeError> {
         return Err(EncodeError::BadLength);
     }
 
-    let len = data.len() * 5 / 4;
+    let len = data.len() * 5 / 4 + 1;
```

Comment:
Overall looks good. Does `zmq_z85_encode` add a trailing null? If so, might be worthwhile adding a note about the extra one byte being read.